runtime/common/addresslocation_test.go

```diff
@@ -65,6 +65,18 @@ func TestAddressLocationTypeID(t *testing.T) {
 	)
 }
 
+func TestAddressLocationID(t *testing.T) {
+
+	t.Parallel()
+
+	location, _, _ := decodeAddressLocationTypeID(nil, "A.0000000000000001.Bar.Baz")
```

Comment:
Maybe also assert there is no error here and below:

```suggestion
	location, _, err := decodeAddressLocationTypeID(nil, "A.0000000000000001.Bar.Baz")
	require.NoError(t, err)
```

Reply:
Good idea!
Fixed in https://github.com/onflow/cadence/pull/2372/commits/4643d37ca193b6a6de72e576aa7505fe401c02b1